aws-operator-role/iam-policy.json

```diff
@@ -139,6 +139,65 @@
                 "iam:CreateServiceLinkedRole"
             ],
             "Resource": "${arn_prefix}:iam::${account_id}:role/aws-service-role/*"
+        },
+	{
+            "Effect": "Allow",
+            "Action": [
+                "iam:CreateOpenIDConnectProvider",
+                "iam:DeleteOpenIDConnectProvider",
+                "iam:ListOpenIDConnectProviderTags",
+                "iam:TagOpenIDConnectProvider",
+                "iam:UntagOpenIDConnectProvider",
+                "iam:ListOpenIDConnectProviders",
+                "iam:GetOpenIDConnectProvider",
+                "cloudfront:TagResource",
+                "cloudfront:UntagResource",
+                "cloudfront:GetCloudFrontOriginAccessIdentity",
+                "cloudfront:CreateCloudFrontOriginAccessIdentity",
+                "cloudfront:DeleteCloudFrontOriginAccessIdentity",
+                "cloudfront:GetDistribution",
+                "cloudfront:CreateDistribution",
+                "cloudfront:UpdateDistribution",
+                "cloudfront:DeleteDistribution",
+                "cloudfront:ListDistributions",
+                "cloudfront:ListTagsForResource"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:CreateBucket",
+                "s3:DeleteBucket",
+                "s3:DeleteObject",
+                "s3:GetBucketLogging",
+                "s3:GetObject",
+                "s3:ListBucket",
+                "s3:PutBucketAcl",
+                "s3:PutBucketEncryption",
+                "s3:PutBucketLogging",
+                "s3:PutBucketLifecycle",
+                "s3:PutBucketTagging",
+                "s3:PutObjectAcl",
+                "s3:PutObject",
+                "s3:PutBucketPolicy",
+                "s3:PutBucketPublicAccessBlock",
+                "s3:PutEncryptionConfiguration",
+                "s3:PutLifecycleConfiguration",
+                "s3:PutObjectAcl"
+            ],
+            "Resource": "arn:aws:s3:::*-g8s-*"
+        },
```

Comment:
you might want to check line 62 and below there are already some S3 permissions

Reply:
removed all duplicates, thank you